cmd/kubelet/app/server.go

```diff
@@ -625,6 +625,7 @@ func CreateAPIServerClientConfig(s *options.KubeletServer) (*restclient.Config,
 		return nil, err
 	}
 
+	clientConfig.UserAgent = restclient.UserAgentForComponent("kubelet")
```

Comment:
so we hard-code this to kubelet, even if somebody renames it to virtlet, somethinglet?

Reply:
also there was the comment about the constant. Why isn't it used?

Reply:
comment please why we have this hard-coding

Reply:
i will use the `componentName` constant. Why would somebody want to rename it to virtlet? ;-)

Reply:
s/rename/fork/